### ISSUE TITLE
[Data Bug] db_fpecr_latest now shows NA for every player's sportsdata_id 

### ISSUE THREAD

**Issue by m-j-g** [tanho63/dynastyprocess#63]
**Describe the data bug**
sportsdata_id

** Which file is having trouble?**
db_fpecr_latest

**Expected data**
sportsdata_id (along with some other ID's) used to not show NA for every player

**Comment by tanho63** [comment]
Yeah, this is a known thing unfortunately. Doesn't look like the fantasypros scrape/api is returning it any longer :(
